src/bacnet/basic/object/bi.c

```diff
@@ -34,11 +39,26 @@ struct object_data {
     bool Present_Value : 1;
     bool Polarity : 1;
     bool Write_Enabled : 1;
+    unsigned Event_State:3;
     uint8_t Reliability;
     const char *Object_Name;
     const char *Active_Text;
     const char *Inactive_Text;
-    const char *Description;
+const char *Description;
```

Comment:
Maybe run clang-format over the file to make the format consistent?  For tables where clang doesn't play nice (i.e. property lists) you can wrap in `/* clang-format off */` and `/* clang-format on */`

Reply:
Apologies for that... Forgot to %=G in my vim ;)

Reply:
Re-worked.